app/presenters/tree_node/middleware_server.rb

```diff
@@ -0,0 +1,5 @@
+module TreeNode
+  class MiddlewareServer < Node
+    set_attribute(:title) { |object| object.name.presence + '-' + object.host_name }
```

Comment:
This is our internal convention, but RuboCop won't mark it wrong as you're using variables...